portal/config/eproms/site_persistence_file.json

```diff
@@ -1696,6 +1696,19 @@
         "HIDE_TRUENTH_ID_FIELD = True\n",
         "TRUENTH_LINK_URL = False\n",
         "MOVEMER_LINK_URL = 'https://movember.com/'\n",
+        "TREATMENT_OPTIONS = [\n",
+        "   {'code': '373818007', 'display': 'Started watchful waiting'},\n",
+        "   {'code': '424313000', 'display': 'Started active surveillance'},\n",
+        "   {'code': '26294005', 'display': 'Radical prostatectomy (nerve-sparing)'},\n",
+        "   {'code': '26294005-nns', 'display': 'Radical prostatectomy (non-nerve-sparing)'},\n",
+        "   {'code': '33195004', 'display': 'External beam radiation therapy'},\n",
+        "   {'code': '228748004', 'display': 'Brachytherapy'},\n",
+        "   {'code': 'androgen deprivation therapy - chemical', 'display': 'Androgen deprivation therapy (ADT) - Chemical'},\n",
+        "   {'code': 'androgen deprivation therapy - surgical orchiectomy', 'display': 'Androgen deprivation therapy (ADT) - Surgical orchiectomy'},\n",
+        "   {'code': 'whole-gland ablation', 'display': 'Whole-gland ablation'},\n",
+        "   {'code': 'focal-gland ablation', 'display': 'Focal-gland ablation'},\n",
+        "   {'code': '999999999', 'display': 'None of the above'}\n",
```

Comment:
Hmm, I don't love this.  Very likely we're going to end up with codes or display values out of sync w/ the database.  This would also need to be translated.  Secondly, a code alone doesn't define a codeable concept - you need the system as well.